contacts.routing.yml

```diff
@@ -28,3 +28,15 @@ contacts.add_org_form:
     _permission: 'add contacts'
   options:
     _admin_route: TRUE
+
+contacts.dash_block_move:
```

Comment:
I wonder if it's worth putting this as a route under the contact_tab entity, so something like `entity.contact_tab.position_block`? Wonder if that would be more intuitive? I def think position block makes more sense than move block.

I also wonder whether we need to have a single POST endpoint that updates all the regions/weights for blocks on a single tab. If you re-arrange the blocks, it wont just be the single block that has a change, but potentially all the others around it. Having the javascript POST all the blocks would go a long way in simplifying that?